modules/azure/availabilityset.go

```diff
@@ -0,0 +1,149 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// AvailabilitySetExists indicates whether the speficied Azure Availability Set exists
```

Comment:
NIT: All the functions that are not `E` should have the following text:

```suggestion
// AvailabilitySetExists indicates whether the speficied Azure Availability Set exists. This function would fail the test if there is an error.
```

Reply:
Fixed all occurrences.